09-workload.md

```diff
@@ -6,12 +6,20 @@ The cluster now has an [Traefik configured with a TLS certificate](./08-secret-m
 
 > :book: The Contoso app team is about to conclude this journey, but they need an app to test their new infrastructure. For this task they've picked out the venerable [ASP.NET Core Docker sample web app](https://github.com/dotnet/dotnet-docker/tree/master/samples/aspnetapp).
 
+1. Customize the host name of the Ingress resource to match your custom domain
+
+   > An example file is provided that will be used later with the built-in kubectl customization tool, Kustomize. This lets you patch the default Ingress resource definition with your own custom values.
+
+   ```bash
+   sed "s/<domain-name>/${DOMAIN_NAME}/g" workload/aspnetapp-ingress-patch.yaml.example > workload/aspnetapp-ingress-patch.yaml
```

Comment:
Part of a kustomization/workflow simplification suggestion (part 4 of 4):

1. Customize the host name of the Ingress resource to match your custom domain. _(Okay to skip this step if domain was left as contoso.com.)_

```suggestion
   sed -i "s/contoso.com/${DOMAIN_NAME}/" workload/aspnetapp-ingress-patch.yaml
```